wagtailio/project_styleguide/templates/patterns/components/streamfields/headline/headline.html

```diff
@@ -0,0 +1,25 @@
+<div class="headline {% if value.dark_background or dark_background %}headline--dark-theme{% endif %}">
+    <div class="grid">
+        {% if value.icon %}
+            <svg class="headline__icon" aria-hidden="true">
+                <use xlink:href="#{{ value.icon }}" />
+            </svg>
+        {% endif %}
+
+        <h2 class="headline__heading heading-one">{{ value.heading }}</h2>
+
+        {% if value.subheading %}
+            <p class="headline__subheading intro-big">{{ value.subheading }}</p>
+        {% endif %}
+
+        {% if value.intro %}
+            <p class="headline__intro">{{ value.intro }}</p>
+        {% endif %}
+
+        {% if value.cta_url or value.cta_page %}
+            <div class="headline__button-wrap">
+                {% include "patterns/components/buttons/button.html" with arrow=True url=url title=value.cta_text classes="headline__button" %}
```

Comment:
I think that this should be provided by the headline, can we do just {% if value.url %}?